content/introduction/_index.md

```diff
@@ -1,60 +1,188 @@
+The most culturally prominent examples are likely file-sharing networks like BitTorrent, 
+and, more recently, the proliferation of blockchain networks that communicate in a 
+peer-to-peer fashion.
 
-While libp2p has several implementations, it is fundamentally a set of protocols for peer identity, discover, routing, transport and more.
+### What problems can libp2p solve?
 
-See the [specifications section](/reference/specs/) for details.
+While peer-to-peer networks have many advantages over the client-server model, 
+there are unique challenges that require careful thought and practice to overcome.
 
-### Implementations
+libp2p lets all users preserve their network identity, overcome network censorship, 
+and communicate over different transport protocols.
 
-At the core of libp2p is a set of [specifications](/reference/specs/), which together form the definition for what libp2p is in the abstract and what makes a "correct" libp2p implementation. Today, implementations of libp2p exist in several languages, with varying degrees of completeness. The most complete implementations are in [Go](/reference/go/), [JavaScript](/reference/js/), and [Rust](https://github.com/libp2p/rust-libp2p).
+In overcoming these challenges while building [IPFS](https://ipfs.io),
+we took care to build our solutions in a modular, composable way into what is 
+now libp2p. Although libp2p grew out of IPFS, it is not dependent on IPFS, and 
+today, [many projects][built_with_libp2p] use libp2p as their networking layer. 
 
-In addition to the implementations above, the libp2p community is actively working on implementations in [python](https://github.com/libp2p/py-libp2p) and [the JVM via Kotlin](https://github.com/web3j/libp2p). Please check the project pages for each implementation to see its status and current state of completeness.
+Together, we can leverage our collective experience and solve these foundational 
+problems in a way that benefits an entire ecosystem of developers and a world of users.
 
+Here, we'll briefly outline the main problem areas that libp2p attempts to address. 
+This is an ever-growing space, so don't be surprised if things change over time. 
+If you notice something missing or have other ideas for improving this documentation, 
+please [reach out to let us know][help_improve_docs].
 
-## Community
+### Data transmission
 
-Get in touch with other members of the libp2p community who are building tools and applications with libp2p! You can ask questions, discuss new ideas, or get support for problems at [libp2p discussion forums](https://discuss.libp2p.io/), but you can also [hop on IRC](/community/irc/) for a quick chat.
+The transport layer is at the foundation of libp2p, which is responsible for 
+the actual transmission and receipt of data from one peer to another. There are many 
+ways to send data across networks in use today, with more in development and still more yet 
+to be designed. 
```

Comment:
Should we mention existing transports (TCP, QUIC, WebSocket, WebTransport, WebRTC)? Might help to make things more concrete.

Reply:
That's good, added for more context.